src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -2151,7 +2151,7 @@ anchor_right = 0.0
 anchor_bottom = 0.0
 margin_top = 45.0
 margin_right = 303.0
-margin_bottom = 107.0
```

Comment:
This file seems to only have margin changes, so maybe reset this to the master branch version?

Reply:
I think these changes is due to the new min size I set for the line charts, and as a result the graphs in the editor gains height a bit thus these margin changes.

Reply:
Ah, I can accept that explanation.